pkg/pillar/cmd/zedagent/handleinfo.go

```diff
@@ -0,0 +1,655 @@
+package zedagent
```

Comment:
Splitting handlemetrics into two isn't a bad idea, but
1. is better done as a separate PR
2. copyright/license is missing
3. the file should really be called reportinfo.go since it doesn't have any handlers for Info messages.

Reply:
+1 for a separate PR

Reply:
https://github.com/lf-edge/eve/pull/1377